eng/pipelines/templates/steps/build-and-test.yml

```diff
@@ -59,7 +56,10 @@ steps:
     displayName: 'Run tests'
     inputs:
       mavenPomFile: pom.xml
-      options: '$(DefaultTestOptions) -DAZURE_TEST_DEBUG=$(IsDebug) -Dparallel-test-playback=${{ parameters.ParallelTestPlayback }} ${{ parameters.TestOptions }} -T ${{ parameters.TestParallelization }}'
+      ${{ if eq(variables['RunAggregateReports'], 'true') }}:
```

Comment:
This will not work, at the time it's evaluated the variable will always be undefined.

Reply:
You may need a prior powershell step that processes all the options and sets them into a final variable that gets consumed by the maven task. Arguably this is easier to maintain than doing all the yaml global stuff anyway at this point.

Reply:
This has been completely reverted as I've been misreading `javadoc.skip` as `jacoco.skip`.